website/src/pages/use-cases/code-health.tsx

```diff
@@ -0,0 +1,366 @@
+                            by="Justin Phillips, Software Engineer at Lyft"
+                            logoImage="/external-logos/lyft-logo.svg"
+                            logoAlt="Lyft"
+                            border={true}
+                            headline="Lyft boosts code health and accelerates developer velocity"
+                            linkText="Read the case study"
+                            link="/case-studies/lyft-monolith-to-microservices"
+                        />
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-bold text-lg-center text-left mb-lg-6 mb-md-6 mb-4">
+                        How Sourcegraph helps
+                    </h1>
+                </div>
+                <div className="pb-lg-5 pb-md-6 pb-5">
+                    <CustomCarousel items={items} autoAdvance={true} smallPanel={true} />
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-gradient-saturn-saturated py-8">
+            <ContentSection>
+                <QuoteCarousel items={quoteCarouselItems} autoAdvance={true} />
+            </ContentSection>
+        </div>
+
+        <div className="bg-light-gray-3 py-7">
+            <ContentSection>
+                <div className="row d-flex flex-column mx-4 mx-lg-0 align-items-lg-center align-items-left">
+                    <div className="mb-5 d-flex flex-column px-lg-7 text-center">
+                        <h1 className="font-weight-bold">Get started with Sourcegraph</h1>
+                        <p>Give your team the tools they need to build a healthier codebase.</p>
+                    </div>
+                    <div className="d-flex flex-column">
+                        <Link
+                            className="btn btn-primary"
+                            to="/demo"
+                            title="Request a Demo."
+                            data-button-style={buttonStyle.primary}
+                            data-button-location={buttonLocation.bodyDemo}
+                            data-button-type="cta"
+                        >
+                            Request a demo
+                        </Link>
+                        <Link to="/use-cases" className="d-flex justify-content-center mt-4">
+                            <p className="font-weight-bold">Explore other use cases</p>
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
```

Comment:
[Duplicate comment](https://github.com/sourcegraph/about/pull/5259/files#r861353322)

Reply:
Thank you, Becca. This one is handled in 3cb96fe!

Reply:
Looks good, thanks!!